src/utils/pgwire/src/pg_protocol.rs

```diff
@@ -337,24 +338,20 @@ where
         let start = Instant::now();
         let session = self.session.clone().unwrap();
         let session_id = session.id().0;
-        let res = self.inner_process_query_msg(sql, session).await;
-        // Record query run successfully
+        let result = self.inner_process_query_msg(sql, session).await;
+
         let mills = start.elapsed().as_millis();
         let truncated_sql = &sql[..std::cmp::min(sql.len(), 1024)];
-        match res {
-            Ok(x) => {
-                tracing::trace!(
-                    target: "pgwire_query_log",
-                    "(simple query) session: {}, status: ok, time: {}ms,  sql: {}", session_id, mills, truncated_sql);
-                Ok(x)
-            }
-            Err(err) => {
-                tracing::trace!(
-                    target: "pgwire_query_log",
-                    "(simple query) session: {}, status: err, time: {}ms,  sql: {}", session_id, mills, truncated_sql);
-                Err(err)
-            }
-        }
+        tracing::trace!(
```

Comment:
I remember we have `release_max_level_debug`. Will the `trace!` be eliminated..?